celavi/pylca_celavi/insitu_emission.py

```diff
@@ -4,7 +4,7 @@
 def preprocessing(year,df_static):
 
     """
-    This function preprocesses the process inventory before the LCA calculation. It joins the dynamic LCA
+    This function preprocesses the emissions inventory before the LCA calculation. It joins the dynamic LCA
```

Comment:
I have changed the docstring for this function. There were some issues that required to be corrected.

Reply:
Please check the comment in the next section. Regarding dummy flows, if for some reason the emissions inventory has a flow that does not match with any process, it will result in the technology matrix not being square resulting in issues with solution. Thus the dummy flows are removed. Its an error checking step.